rare/components/tabs/games/game_info/game_settings.py

```diff
@@ -133,6 +134,9 @@ def __init__(self, core: LegendaryCore, parent):
             self.linux_settings_scroll.setVisible(False)
         self.game_settings_layout.setAlignment(Qt.AlignTop)
 
+        self.EnvVars = EnvVars(self)
+        self.linux_settings_contents_layout.insertWidget(0, self.EnvVars)
```

Comment:
Add it to all settings. And variable names are snake case (self.env_vars)

Reply:
Changed to snake case.
Different setting/layout will be done in a different PR? Probably